### ISSUE TITLE
No documentation generated with Xcode 11 GM 1 and 2, Jazzy 0.11.1

### ISSUE THREAD

**Issue by tyten** [realm/jazzy#1108]
# Scenario
- macOS: 10.14.6
- Xcode: 11 GM 1 _or_ 2
- Jazzy: 0.11.1

## Command
`bundle exec jazzy -c -o documentation --source-directory=SUBFOLDER\ with\ spaces`

# Result
## Xcode 11
`xcodebuild` runs, with 0% coverage, documentation generated but nothing included.

## Xcode 10.3
Behaves as expected

**Comment by jpsim** [comment]
Can you share the directory contents for where this is being run?

**Comment by tyten** [comment]
Without going into too much detail publicly, this framework repo has folder structure as follows (All Swift):
```
Root (where command is run)
⎣.git
⎣.gitignore
⎣Gemfile&lock
⎣README.md
⎣build
⎣documentation folder
⎣Demo App (folder)
⎣*Framework Folder* (--source-directory points here)
  ⎣build
  ⎣Framework Source Folder
    ⎣Components
    ⎣Extensions
    ⎣Protocols
    ⎣Resources
    ⎣Supporting Files
    ⎣View Controllers
  ⎣Framework.xcodeproj
  ⎣FrameworkTests Folder
⎣.xcworkspace
⎣Demo App.xcodeproj
⎣Tests Folder
⎣Random Other Folders
```

**Comment by johnfairh** [comment]
I think the workaround is to add `-x USE_SWIFT_RESPONSE_FILE=NO` to the jazzy args - there's a problem with our treatment of Xcode 11 swift response files with spaces (as you maybe hinted).

**Comment by strike65** [comment]
I have the same problem. After updating to Xcode 11 Jazzy doesn't work anymore. Unfortunately the tip doesn't help me.  I am not even able to update to 0.11.1. I get the error message:
```bash
ERROR: Error installing jazzy:
ERROR: Failed to build gem native extension.
mkmf.rb can't find header files for ruby at /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/include/ruby.h
```
Do you have any ideas?
-- 
System: 
macOS 10.14.6
Ruby: 2.3.7
gem: 3.0.6
Xcode: 11 (11A420a)

**Comment by johnfairh** [comment]
@strike65 this is a different problem: you are using the Mojave system Ruby (2.3) but Xcode 11 has the development files for Catalina (2.6).  Either switch back to Xcode 10.x to do the install or install an independent Ruby (eg. `rbenv` via carthage).

**Comment by tyten** [comment]
I can confirm that added flag makes it work with Xcode 11. Thanks so much.

**Comment by tyten** [comment]
Although it does look like it ended up ignoring an enum in one of my source files?

**Comment by tyten** [comment]
In this particular case, enums that are declared in a source file that also contains a class definition were documented. Enums that sat in their own source file were ignored.

**Comment by tyten** [comment]
According to the output while running, Jazzy didn't even parse it. Is parsed in Xcode 10.3 command line tools _with_ the `-x` flag

**Comment by johnfairh** [comment]
Closing this one: main problem of no files at all being used when there are special characters in the path is fixed in master and 0.11.2.  If the 'one file being ignored' problem persists we should track that in a new issue.

**Comment by dareknowak** [comment]
It's also worth to mention, that some of users might have missing headers. See: https://github.com/castwide/vscode-solargraph/issues/78#issuecomment-535854595
